ixmp/backend/base.py

```diff
@@ -81,6 +81,46 @@ def get_nodes(self):
         set_node
         """
 
+    @abstractmethod
+    def get_timeslices(self):
+        """Return all subannual timeslices defined on the Platform instance.
+
+        Yields
+        -------
+        tuple
+            The members of each tuple are:
+
+            ========= =========== ===
+            ID        Type        Description
+            ========= =========== ===
+            name      str         Time slice name
+            category  str         Time slice category
+            duration  float       Time slice duration (fraction of year)
```

Comment:
should that be float or double?

Reply:
I guess that's just referring to python float type (not indicating single or double precision). See other method docs in same file

Reply:
ok, thanks